docs/src/openapi-v1.yaml

```diff
@@ -2048,6 +2180,119 @@ components:
+        spans:
+          type: array
+          items:
+            type: object
+            description: |
+              Span is compromised of id, name, parentId, target, and wasm.
+        storageKeys:
+          type: string
+          description: Hex encoded storage keys used to filter events.
+        tracingTargets:
+          type: string
+          description: Targets used to filter spans and events.
+    SpanId:
+      type: object
+      properties:
+        name: 
+          type: string
+        target:
+          type: string
+        id:
+          type: string
+          format: unsignedInteger
+    OperationAmountCurrency:
+      type: object
+      properties:
+        symbol:
+          type: string
+          example: KSM
+    OperationAmount:
+      type: object
+      properties:
+        values:
+          type: string
+          format: unsignedInteger
+        currency:
+          $ref: '#/components/schemas/OperationAmountCurrency'
+    OperationPhase:
+      type: object
+      properties:
+        variant:
+          type: string
+          enum:
+          - onInitialize
+          - initialChecks
+          - applyExtrinsic
+          - onFinalize
+          - finalChecks
+          description: Phase of block execution pipeline.
+        extrinsicIndex:
+          type: string
+          format: unsignedInteger
+          description: | 
+            If phase variant is `applyExtrinsic` this will be the index of 
+            the extrinsic.
+    Operation:
+      type: object
+      properties:
+        phase:
+          $ref: '#/components/schemas/OperationPhase'
+        parentSpanId:
+          $ref: '#/components/schemas/SpanId'
+        primarySpanId:
+          $ref: '#/components/schemas/SpanId'
+        eventIndex:
+          type: string
+          format: unsignedInteger
+          description: Index of the underlying trace event.
+        address:
+          type: string
+          description: |
+           Account this operation affects. Note - this will be an object like
+           `{ id: address }` if the network uses `MultiAddress`
+        storage:
+          type: object
+          properties:
+            pallet:
+              type: string
+            item:
+              type: string
+            field1:
+              type: string
+              description: |
+                A field of the storage item. (i.e `system::Account::get(address).data`)
+            field2:
+              type: string
+              description: |
+                A field of the struct described by field1 (i.e
+                `system::Account::get(address).data.free`)
+        amount:
+          $ref: '#/components/schemas/OperationAmount'
+    BlocksTraceOperations:
+      type: object
+      properties:
+        at:
+          $ref: '#/components/schemas/BlockIdentifiers'
+        operations:
+          type: array
+          items:
+            $ref: '#/components/schemas/Operation'
```

Comment:
Note: intentionally not including action group in the response. We should discuss if we think action groups should be available at all and if we want to make any guarantees about their stability.